oidcprovider/opkclient.go

```diff
@@ -1,8 +1,7 @@
-package parties
+package oidcprovider
```

Comment:
Could it be called `providers`?

Reply:
or `client`?

Reply:
Yeah that's better, or maybe `provider` (never sure whether singular or plural is better). I wanted to call this `oidc` but it was annoying to conflict with the https://github.com/zitadel/oidc library

Reply:
A simple noun is better than an acronym, plural makes sense to me in this case, but it is not a big deal. `client` is okay to me, but it's super ambiguous and if we have another option, I'd go with that, which we do in this case.

Reply:
I would argue that technically none of the objects in this package are oidc providers, but modified oidc clients used for enacting the opk protocol against oidc providers. `oidc` seems better than `provider` here.

Reply:
I think it's okay to call it a provider since it's a client-side representation of a provider.
I doubt this library would ever include an implementation of a full-blown OIDC server.
In any case, I'm okay with `client`, but I think it's a bit silly to use the acronym OIDC through a library that mostly deals with OIDC.

Reply:
sorry, I know this is harping on a simple naming issue for too long but...now that I've been neck-deep in this code for a little, what do you guys think about this?
```
.
├── client
    └── opkclient.go
    └── providers
        ├── google.go
        └── github.go
```

Reply:
Apologies for the delay getting back on this! This LGTM, will update